components/atoms/SearchBar.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types";
+
+/**
+ *  component
+ */
+export function SearchBar(props) {
+  return (
+    <div className="inline-flex">
+      <form className="inline-flex py-0.5">
+        <input
+          type="text"
+          placeholder={props.placeholder}
+          className="border border-custom-gray-light text-search-text-gray focus:outline-none"
+        />
+
+        <button
```

Comment:
So rather than this being a form and having a submit button, we just need to pass a callback when the button is clicked to know when someone wants to search